apps/mark-scan/backend/src/custom-paper-handler/state_machine.ts

```diff
@@ -76,6 +76,7 @@ type PaperHandlerStatusEvent =
   | { type: 'VOTER_INITIATED_PRINT'; pdfData: Buffer }
   | { type: 'PAPER_IN_OUTPUT' }
   | { type: 'PAPER_IN_INPUT' }
+  | { type: 'CONFIRM_INVALIDATE_BALLOT' }
```

Comment:
Maybe rename this `VOTER_CONFIRMED_INVALIDATED_BALLOT` to be consistent with the other voter actions?

Reply:
[ac47292](https://github.com/votingworks/vxsuite/pull/3962/commits/ac472921d2a0ae8222d0827d0ab717ae927c667a)